### PR TITLE
[877109] Added the API links, animation, right tapped event and right tap command UG content.

### PR DESCRIPTION
**Pull request by AneesFathimaS**
Added the API links for new feature and control.
Added the UG content for Right tapped event and right tap command in treeView and ListView.
Added animation UG content in treeview.
Added the dismiss popup UG content.

### CODE REVIEW

**Review by Lakshminatarajan — APPROVED** [review]
Approved

File: MAUI/Chat/data-binding.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ documentation: ug
 
 # Data Binding in .NET MAUI Chat (SfChat)
 
-The `SfChat` control allows to bind any existing collection of data objects as message collection using the `ItemSource` and `ItemsSourceConverter` property as follows.
+The `SfChat` control allows to bind any existing collection of data objects as message collection using the [ItemSource](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Chat.SfChat.html#Syncfusion_Maui_Chat_SfChat_ItemsSource) and [ItemsSourceConverter](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Chat.SfChat.html#Syncfusion_Maui_Chat_SfChat_ItemsSourceConverter) property as follows.
```
Comment (PiruthivirajM) [inline MAUI/Chat/data-binding.md:12]: API name is [ItemsSource]

File: MAUI/ListView/working-with-sflistview.md [hunk 2]
```diff
@@ -96,6 +96,31 @@ The [ItemDoubleTapped](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ListV
...
+
+    private void ListView_ItemRightTapped(object sender, Syncfusion.Maui.ListView.ItemRightTappedEventArgs e)
+    {
+        DisplayAlert("Item RightTapped", "ListView item right tapped", "Close");
+    }
+
+{% endhighlight %}
+{% endtabs %}
+
+The [ItemRightTapped](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ListView.SfListView.html#Syncfusion_Maui_ListView_SfListView_ItemRightTapped) event is used for the following use cases:
```
Comment (PiruthivirajM) [inline MAUI/ListView/working-with-sflistview.md:120]: ignore this point.

**Review by PiruthivirajM — COMMENTED** [review]
resolve the comments

**Review by PiruthivirajM — APPROVED** [review]

**Pull request merged**